runtime/src/lib.rs

```diff
@@ -428,6 +431,99 @@ impl Contains<Call> for BaseFilter {
     }
 }
 
+/// Filter the calls that can be used as free calls.
+pub struct FreeCallsFilter;
+impl Default for FreeCallsFilter { fn default() -> Self { Self } }
+impl Contains<Call> for FreeCallsFilter {
+    fn contains(c: &Call) -> bool {
+        match *c {
+            Call::Spaces(..) => true,
+            Call::SpaceFollows(..) => true,
+            Call::ProfileFollows(..) => true,
+            Call::Posts(..) => true,
+            Call::Reactions(..) => true,
+            Call::System(..) => cfg!(feature = "runtime-benchmarks"),
+            _ => false,
+        }
+    }
+}
+
+/// A calculation strategy for free calls quota
+pub struct FreeCallsCalculationStrategy;
+impl Default for FreeCallsCalculationStrategy { fn default() -> Self { Self } }
+impl pallet_free_calls::QuotaCalculationStrategy<Runtime> for FreeCallsCalculationStrategy {
+    fn calculate(
+        current_block: <Runtime as frame_system::Config>::BlockNumber,
+        locked_info: Option<LockedInfoOf<Runtime>>
+    ) -> Option<NumberOfCalls> {
+        locked_info.and_then(|locked_info| {
+            if current_block >= locked_info.unlocks_at {
+                None
+            } else {
+                // TODO: add more sophisticated calculation
+                // TODO: think if we should make NumberOfCalls -> u32 instead of u16
+                Some((locked_info.locked_amount / 11 /*decimals*/) as NumberOfCalls)
```

Comment:
Should be changed to Some((locked_info.locked_amount / 100_000_000 /*decimals*/) as NumberOfCalls)